systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -183,6 +184,27 @@ void testKafkaAndZookeeperScaleUpScaleDown() {
         assertNoCoErrorsLogged(TimeMeasuringSystem.getDurationInSecconds(testClass, testName, operationID));
     }
 
+    @Test
+    @Tag(REGRESSION)
+    void testEODeletion () {
```

Comment:
Checkstyle does not like an empty space before `()`

Reply:
Thanks, fixed!